### PR TITLE
basic model inference outline, still need to add calls in api.py

### PR DESCRIPTION
**Pull request by minneker**
Basic functionality for model inference using EEGNet. Once we know the interface for reading data from the file, we can pass it to these functions in `api.py`.

### CODE REVIEW

**Review by sakethn-1764585 — APPROVED** [review]
Looks good, is everything after constants.py lifted from EEGNet?

Also does the model work?

**Review by saketh-n — APPROVED** [review]
Lol approved with my alt.

Unfamiliar with what offline_training.ipynb does, but if it's from EEGNet I'll trust it to be good.

File: api/api.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 cors = CORS(app)
 app.config['CORS_HEADERS'] = 'Content-Type'
 
+
```
Comment (saketh-n) [inline api/api.py:9]: Why'd you add extra line here and delete one at the end?

Comment (minneker) [reply api/api.py:9]: PEP standards expect 2 blank lines before a decorator and a blank line at the end of the file. My PyCharm IDE highlighted them.

Comment (minneker) [reply api/api.py:9]: PEP standards expect 2 blank lines before a decorator and a blank line at the end of the file. My PyCharm IDE highlighted them.

**Comment by minneker** [comment]
@saketh-n No worries on the alt account haha. I should add some tests for the model, good point. The `offline_training.ipynb` is a skeleton Jupiter notebook where I'll add the offline transfer learning. In my next PR I'll add the offline training as well as some model unit tests!

File: api/api.py (hunk 1 above)
Comment (saketh-n) [reply api/api.py:9]: Sounds good

**Pull request merged**